### DISCUSSION TITLE
Link to / share this page

### DISCUSSION THREAD

**Discussion by JayFarei** [roam-garden/i#60] [Feature requests & Ideas]
Expand on: https://github.com/roam-garden/i/discussions/59 and https://github.com/roam-garden/i/discussions/38 for a contextual menu in stacked pages - in this case I suggest to introduce a "link to" or "share" this page.

UX: 
After selecting that option - a notification confirms that the link has been copied to the clipboard.

Details:
Generate a URL based on the page name where the action was invoked.

Example: 
- A user navigates to the page `The Best Page` - URL: https://user.roam.garden/?stackedPages=%2FThe-Best-Page
- User selects the link to this page action
- The action generate this URL: https://user.roam.garden/The-Best-Page
- The URL is copied to the clipboard
- The user is notified that the action completed

**Comment by Stvad** [comment]
I think this duplicates https://github.com/roam-garden/i/discussions/39 🙂  
Thanks for a detailed UX description though!

**Reply by JayFarei** [reply]
missed that! +1 to that :)

**Reply by Stvad** [reply]
ok closing this in favor of that one :)
